src/backend/controllers/battleController.js

```diff
@@ -219,6 +211,41 @@ exports.reportResults = async (req: Request, res: Response) => {
 					.json({ msg: 'Could not find userOne'});
 			}
 
+			// Update money with first win of day bonus, if applicable
+			const aDay = 60 * 60 * 24 * 1000;
+			const compare = (!userOne.stats.lastFirstWinOfDay) ? (new Date() - userOne.stats.lastFirstWinOfDay) : 0;
+			// If last recorded first win of the day is null or it's been over a day since
+			if (userOne.stats.lastFirstWinOfDay == null || compare > aDay) {
+				// If it's been over 2 days since, the streak is broken
+				if (compare > (aDay*2)) {
+					userOne.stats.firstWinOfDayStreak = 0;
+					console.log("Streak Broken :(");	
+				}
+				// Give bonus based on current streak
+				if (userOne.stats.firstWinOfDayStreak === 0) {
+					// +100 for first day of the streak
+					userOne.money += (battle.prizeMoney + 100);
+					userOne.stats.firstWinOfDayStreak++;
+					console.log("Given First Win of Day 1 Bonus");
+				}
+				else if (userOne.stats.firstWinOfDayStreak === 1) {
+					// +200 for second day of the streak
+					userOne.money += (battle.prizeMoney + 200);
+					userOne.stats.firstWinOfDayStreak++;
+					console.log("Given First Win of Day 2 Bonus");					
+				}
+				else if (userOne.stats.firstWinOfDayStreak === 2) {
+					// +300 for third day of the streak, and streak resets
+					userOne.money += (battle.prizeMoney + 300);
+					userOne.stats.firstWinOfDayStreak = 0;
```

Comment:
I would keep the streak at 3 days. It seems kind of silly to lose your streak despite winning

Reply:
So you mean even after 3 days maintain the 300 per day? That's how I'm interpreting what you're saying. If that's the case that's an easy logic fix then.